internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -239,6 +244,20 @@ func (k *KafkaManager) calculateCapacityByRegionAndInstanceTypeForManualClusters
 		return totalUsed, instanceTypeUsed
 	}
 
+	// enterprise data plane clusters
+	for _, cluster := range streamingUnitsByRegion {
+		if cluster.ClusterType == api.EnterpriseDataPlaneClusterType.String() {
+			result = append(result, services.KafkaStreamingUnitCountPerCluster{
+				Region:        cluster.Region,
+				InstanceType:  cluster.InstanceType,
+				ClusterId:     cluster.ClusterId,
+				Count:         int32(cluster.Count),
```

Comment:
```suggestion
				Count:         int32(cluster.MaxUnits - cluster.Count),
```

I think this count is meant to represent `available` streaming unit Count.

Reply:
e.g here https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/23d0d0031fd3e60848740dbd5cafad45afff232a/internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go#L299